crates/eframe/src/native/run.rs

```diff
@@ -270,11 +270,11 @@ fn run_and_exit(event_loop: EventLoop<UserEvent>, mut winit_app: impl WinitApp +
             // Platform-dependent event handlers to workaround a winit bug
             // See: https://github.com/rust-windowing/winit/issues/987
             // See: https://github.com/rust-windowing/winit/issues/1619
-            winit::event::Event::RedrawEventsCleared if cfg!(windows) => {
+            winit::event::Event::RedrawEventsCleared if cfg!(target_os = "windows") => {
```

Comment:
I suspect we could remove this special handling of `RedrawEventsCleared` here and elsewhere, since it has effectively been disabled for a long time due to the `if cfg!(windows)` mistake. Maybe someone on Windows can test and verify?

Reply:
Let's remove it when we update winit - the event loop is completely rewritten in the next winit release